src/components/UiLibrary.js

```diff
@@ -14,94 +15,136 @@ import ViewWithFooter from "./SharedComponents/ViewWithFooter";
+          <Button className="my-2 grow ml-3" level="focus" text="RIGHT" />
+        </View>
 
-      <Text className="text-xl">Multiple Buttons Without Focus</Text>
-      <View className="flex-row">
-        <Button className="my-2 grow" text="LEFT" />
-        <Button className="my-2 ml-3 grow" text="RIGHT" />
-      </View>
+        <Text className="text-xl">Multiple Buttons Without Focus</Text>
+        <View className="flex-row">
+          <Button className="my-2 grow" text="LEFT" />
+          <Button className="my-2 ml-3 grow" text="RIGHT" />
+        </View>
 
-      <Text className="text-lg">AddObsButton</Text>
-      <Text>
-        You probably don't want to tap this because you can't escape the
-        modal. Probably need to refactor to separate form from function.
-      </Text>
-      <AddObsButton />
+        <Text className="text-lg">AddObsButton</Text>
+        <Text>
+          You probably don't want to tap this because you can't escape the
+          modal. Probably need to refactor to separate form from function.
+        </Text>
+        <AddObsButton />
 
-      <Text className="text-lg">EvidenceButton</Text>
-      <View className="flex flex-row justify-between">
-        <View>
-          <Text className="text-center">Default</Text>
-          <EvidenceButton />
-        </View>
-        <View>
-          <Text className="text-center">Disabled</Text>
-          <EvidenceButton disabled />
+        <Text className="text-lg">EvidenceButton</Text>
+        <View className="flex flex-row justify-between">
+          <View>
+            <Text className="text-center">Default</Text>
+            <EvidenceButton />
+          </View>
+          <View>
+            <Text className="text-center">Disabled</Text>
+            <EvidenceButton disabled />
+          </View>
+          <View>
+            <Text className="text-center">With Icon</Text>
+            <EvidenceButton icon="photo-camera" />
+          </View>
         </View>
-        <View>
-          <Text className="text-center">With Icon</Text>
-          <EvidenceButton icon="photo-camera" />
-        </View>
-      </View>
-
-      <Text className="text-lg">SecondaryCTAButton</Text>
-      <SecondaryCTAButton>
-        <Text>SecondaryCTAButton</Text>
-      </SecondaryCTAButton>
-      <SecondaryCTAButton disabled>
-        <Text>Disabled SecondaryCTAButton</Text>
-      </SecondaryCTAButton>
 
-      <Text className="text-lg">More Stuff!</Text>
-      <Text className="h-[100px]">
-        Useless spacer at the end because height in NativeWind is confusing.
-      </Text>
-    </ScrollView>
-  </ViewWithFooter>
-);
+        <Text className="text-lg">SecondaryCTAButton</Text>
+        <SecondaryCTAButton>
+          <Text>SecondaryCTAButton</Text>
+        </SecondaryCTAButton>
+        <SecondaryCTAButton disabled>
+          <Text>Disabled SecondaryCTAButton</Text>
+        </SecondaryCTAButton>
+        <Text className="text-lg">Custom iNaturalist Icons</Text>
```

Comment:
This section is more about how to use IconButton, which is definitely useful, but maybe call it "IconButton w/ Custom iNaturalist Icons"?

Also, it would be cool to see all our icons here. If you export `glyphMap` from INatIcon.js, you could map over that to render an `<INatIcon />` for every icon.

Reply:
Noting that we'll need to tweak either the `./node_modules/.bin/generate-icon` script, the `createIconSet` script from `react-native-vector-icons` script, or both to be able to export `glyphMap` automatically, instead of adding this every time a new INatIcon.js component is generated. In the spirit of timeboxing, I'm inclined to make a separate issue for that, as well as renaming the `.ttf` file, which is the other manual change we need to make in this component.